src/Data/Form.purs

```diff
@@ -184,6 +193,19 @@ eval get page = do
           , value
           }
         )
+    DateTime input -> do
+      default <- traverse (evalExpr get) input.default
+      placeholder <- evalExpr get input.placeholder
+      required <- evalExpr get input.required
+      value <- traverse (evalExpr get) input.value
+      pure
```

Comment:
An example of the problem with my validation PR, merging that in after this, we'd easily miss that this isn't being validated